_includes/contact_form.html

```diff
@@ -425,10 +425,14 @@ <h3>{{ site.data.[page.lang].settings.contact_page.support_request_form.title }}
       site.data.[page.lang].settings.contact_page.support_request_form.help_text }}
     </p>
 
+    <b>
+      {{ site.data.[page.lang].settings.contact_page.support_request_form.required_text }} (<abbr title="{{ site.data.[page.lang].settings.contact_page.required }}" class="usa-hint usa-hint--required">*</abbr>).
+    </b>
```

Comment:
Semantically I think this could warrant using a `<strong>` tag instead of `<b>` ?

* https://developer.mozilla.org/en-US/docs/Web/HTML/Element/strong

```suggestion
    <strong>
      {{ site.data.[page.lang].settings.contact_page.support_request_form.required_text }} (<abbr title="{{ site.data.[page.lang].settings.contact_page.required }}" class="usa-hint usa-hint--required">*</abbr>).
    </strong>
```